doc/packages/openfoam-com.section.md

```diff
@@ -0,0 +1,26 @@
+# OpenFOAM (com) {#sec-openfoam-com}
+
+Unfortunately, OpenFOAM binaries assume you execute them via `bash`, after sourcing `$OPENFOAM_DIR/etc/bashrc`.
+To avoid this, binaries from the derivation are wrapped so that they are executed with these requirements.
+
```

Comment:
```suggestion
```


This is an implementation detail users don't need to care about. Unless there's something users need to do here?

Reply:
I thought it could be useful to document it for users already familiar with OpenFOAM.

Reply:
Hm, how would that impact user interaction? I imagine they'd still just run the command, and actually have to do less (i.e. don't have to manually ensure some files are in place somewhere). Maybe it would would help to make explicit in the phrasing that if one is used to openFOAM on traditional Linux distributions, one has to (not) do XYZ?